Sources/python/SimEx/Utilities/wpg_to_opmd.py

```diff
@@ -0,0 +1,88 @@
+import sys
+import numpy
+import math
+import h5py
+import os
+
+from SimEx.Utilities import OpenPMDTools as opmd
+
+def _convertToOPMD(h5_in):
+    """ Take native wpg output and rewrite in openpmd conformant way. """
```

Comment:
we could add some doc strings explaining what type `h5_in` is expected to have (list of strings with file names?):

```python
"""
Take native wpg output and rewrite in openpmd conformant way.

Parameters
----------
h5_in : ...
````